boa_engine/src/bytecompiler/statement/try.rs

```diff
@@ -162,4 +163,36 @@ impl ByteCompiler<'_, '_> {
 
         self.patch_jump(do_not_throw_exit);
     }
+
+    /// Compile a catch or finally block.
+    ///
+    /// If the block contains a break or continue as the first statement,
+    /// the return value is set to undefined.
+    /// See the [ECMAScript reference][spec] for more information.
+    ///
+    /// [spec]: https://tc39.es/ecma262/#sec-try-statement-runtime-semantics-evaluation
+    fn compile_catch_finally_block(&mut self, block: &Block, use_expr: bool) {
+        let mut b = block;
+
+        'outer: loop {
```

Comment:
there is only one loop in so i think we can remove `'outer` label, right?

Reply:
Fixed :)